app/Http/Controllers/ProjectController.php

```diff
@@ -25,9 +27,7 @@ public function listProjects()
      */
     public function view(Project $project)
     {
-        $project->load('comments.user');
-
-        return view('project.view', ['project' => $project]);
+        return view('project.view', ['project' => $project, 'isUserFollower' => $project->isUserFollower(), 'followerCount' => $project->getFollowerCount()]);
```

Comment:
Passing `isUserFollower` and `followerCount` as variables to the view seems redundant to me. `project` will be accessible in the view, so why not just do `$project->isUserFollower()` in the view itself? Idk what best practice is, but that makes more sense to me